### PR TITLE
[torch_glow] Add support for Abs op

### PR DESCRIPTION
**Pull request by dangus36**
Summary:
Added support for abs op in PyTorchModelLoader.

Test Plan:
Added multiple unit tests in abs_test.py which cover basic cases and cases with multidimensional tensors.

### CODE REVIEW

File: torch_glow/tests/nodes/abs_test.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    def __init__(self):
+        super(SimpleAbsModule, self).__init__()
+
+    def forward(self, a):
+        return torch.abs(a + a)
+
+
+class TestAbs(unittest.TestCase):
+    def test_abs_basic(self):
+        """Basic test of the PyTorch cat Node on Glow."""
```
Comment (842974287) [inline torch_glow/tests/nodes/abs_test.py:17]: nit: 
```suggestion
        """Basic test of the PyTorch abs Node on Glow."""
```

**Review by 842974287 — COMMENTED** [review]
Thanks for adding this! Just one typo and I think we're good to go.

File: torch_glow/tests/nodes/abs_test.py (hunk 1 above)
Comment (dangus36) [reply torch_glow/tests/nodes/abs_test.py]: Thanks! Amended commit with changes.

**Review by 842974287 — APPROVED** [review]

File: torch_glow/tests/nodes/abs_test.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from tests import utils
+import unittest
```
Comment (842974287) [inline torch_glow/tests/nodes/abs_test.py:4]: There're some internal linter complaints prevent landing. Would you mind apply my suggestion changes and I'll import again afterwards. Sorry for the inconvenience.

```suggestion

import unittest

import torch
from tests import utils
```

Comment (dangus36) [reply torch_glow/tests/nodes/abs_test.py]: No worries!

**Pull request closed without merging**